### PR TITLE
Profile Detail View UI 

### PR DESCRIPTION
**Pull request by Meghana-12**
Added the UI of Profile detail view, issue: #743. 

Ignitus (1)

### CODE REVIEW

File: src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import { StyledHeading2, StyledHeading4, StyledHeading6 } from '../../../ignitus-UserInterfaceBook/styles';
+import * as C from '../styles';
```
Comment (Paarmita) [inline src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx:3]: We use `C` generally for importing colors, can you please user `P` for Profile

File: src/ignitus-Profile/ignitus-StudentProfile/styles.ts [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  background: ${C.White};
+  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
+  border-radius: 30px;
+  padding-left: 5rem;
+  padding-right: 0;
+  padding-bottom: 5rem;
+`;
+
+export const Container2 = styled.div`
+  margin-right: -0.3rem;
```
Comment (Paarmita) [inline src/ignitus-Profile/ignitus-StudentProfile/styles.ts:24]: Don't use margin in negative.

File: src/ignitus-Profile/ignitus-StudentProfile/styles.ts [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+`;
+
+export const ElementContainer = styled(Container)`
+  max-width: ${minimumWidthQuery[3]};
+  flex:1;
+
+`;
+
+export const HeadingContainer = styled(flexibleRowDiv)`
+  // padding-top: 3rem;
```
Comment (Paarmita) [inline src/ignitus-Profile/ignitus-StudentProfile/styles.ts:44]: Remove comments

File: src/ignitus-Profile/ignitus-StudentProfile/styles.ts [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+`;
+
+export const ContriContent = styled.div`
+  display: flex;
+  flex-direction:row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+`;
+
+export const ContriElement = styled(flexibleRowDiv)`
```
Comment (Paarmita) [inline src/ignitus-Profile/ignitus-StudentProfile/styles.ts:124]: Please try to use understandable convention names instead of `Contri`

Comment (Meghana-12) [reply src/ignitus-Profile/ignitus-StudentProfile/styles.ts:124]: sure, I'll edit that

File: src/ignitus-Profile/ignitus-StudentProfile/styles.ts (hunk 3 above)
Comment (Meghana-12) [reply src/ignitus-Profile/ignitus-StudentProfile/styles.ts:44]: okay

File: src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx (hunk 1 above)
Comment (Meghana-12) [reply src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx:3]: sure!

File: src/ignitus-Profile/ignitus-StudentProfile/styles.ts (hunk 2 above)
Comment (Meghana-12) [reply src/ignitus-Profile/ignitus-StudentProfile/styles.ts:24]: okay

File: src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+import {AppIcon} from '../../../ignitus-Shared/types/iconsTypes/iconEnums';
+import {
+  ButtonBottomRight,
+  RoundedButton,
+} from '../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/buttons';
+import {AboutContent} from '../styles';
+
+
+const ProfileDetailView = () => (
+  <P.Container1>
```
Comment (divyanshu-rawat) [inline src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx:13]: Please don't name styles with suffix 1,2,3.
You can name this one only Container is the top most one.

Comment (Meghana-12) [reply src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx:13]: okay, I'll edit that.

**Comment by divyanshu-rawat** [comment]
@Meghana-12 please remove the .png image added as well, and use the avatar I suggested., please delete the Image.

File: src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+        batch="2015 – 2019"
+      />
+
+      <P.ResearchSection>
+        <HeadingIconContent heading="Research Fields" icon={AppIcon.LocalLibraryIcon}/>
+        <P.ContentContainer>
+          <RoundedButton size="medium" category="grey">Psycholinguistics </RoundedButton>
+          <RoundedButton size="medium" category="grey">Modernist Literature  </RoundedButton>
+        </P.ContentContainer>
+      </P.ElementContainer>
```
Comment (divyanshu-rawat) [inline src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx:38]:
```suggestion
      </P. ResearchSection >
```

**Pull request merged**